server/public/styles.css

```diff
@@ -0,0 +1,5 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
Is this CSS all necessary? `box-sizing` is already set by Bootstrap. What exactly do you want `margin: 0` on? Typically if you want to influence all of the text, you'd just set the `body` font. Are you sure you need to?

Reply:
Resolved in c2d14c1.